lib/hider/accounts.ex

```diff
@@ -37,6 +37,14 @@ defmodule Hider.Accounts do
   """
   def get_user!(id), do: Repo.get!(User, id)
 
+  def get_user_by_cpf(cpf) do
```

Comment:
Generate the hash for the search (MUST BE EQUAL) and search the blind index that match.

It's a good idea to normalize the data before indexing, so you can have better matchs

```
Lucas -> lucas
851.863 -> 851863
```